src/protocolsupport/protocol/utils/types/NetworkEntity.java

```diff
@@ -115,7 +124,29 @@ public void setPeBaseFlag(int bitpos, long value) {
 		public void setPeBaseFlags(long peBaseFlags) {
 			this.peBaseFlags = peBaseFlags;
 		}
-		
+
+		public void setHelmet(ItemStackWrapper helmet) {
+			this.helmet = helmet;
+		}
+
+		public ItemStackWrapper getHelmet() { return this.helmet; }
+
+		public void setChestplate(ItemStackWrapper chestplate) { this.chestplate = chestplate; }
+
+		public ItemStackWrapper getChestplate() { return this.chestplate; }
+
+		public void setLeggings(ItemStackWrapper leggings) {
+			this.leggings = leggings;
+		}
+
+		public ItemStackWrapper getLeggings() { return this.leggings; }
+
+		public void setBoots(ItemStackWrapper boots) {
+			this.boots = boots;
+		}
+
```

Comment:
Blame IntelliJ, sometimes it likss to add a new line for no reason at all, I will fix that soon